optional-container-engine/README.md

```diff
@@ -0,0 +1,73 @@
+# Deploy Bookshelf to Container Engine/Kubernetes
+
+This optional tutorial will walk you through how to deploy the Bookshelf sample application to [Google Container Engine](https://cloud.google.com/container-engine/). This tutorial is also applicable to [Kubernetes](http://kubernetes.io/) outside of Container Engine, but may require additional steps for external load balancing.
+
+## Pre-requisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com).
+
+2. [Enable billing](https://console.cloud.google.com/project/_/settings) for your project. 
+
+3. Install the [Google Cloud SDK](https://cloud.google.com/sdk)
+
+        $ curl https://sdk.cloud.google.com | bash 
+        $ gcloud init
+
+4. Install [Docker](https://www.docker.com/).
+
+## Create a cluster
+
+Create a cluster for the bookshelf application:
+
+    gcloud container clusters create bookshelf \
+        --scope "https://www.googleapis.com/auth/userinfo.email","cloud-platform"
+
+The scopes specified in the `--scope` argument allows nodes in the cluster to access Google Cloud Platform APIs, such as the Cloud Datastore API.
+
+## Create a Cloud Storage bucket
+
+The bookshelf application uses [Google Cloud Storage](https://cloud.google.com/storage) to store image files. Create a bucket for your project:
+
+    $ gsutil mb gs://<your-project-id>
+    $ gsutil defacl set public-read gs://<your-project-id>
+
+## Update config.py
+
+Modify config.py and enter your Cloud Project ID into the `PROJECT_ID` and `CLOUD_STORAGE_BUCKET` field. The remaining configuration values are only needed if you wish to use a different database or if you wish to enable log-in via oauth2, which requires a domain name.
+
+## Build the bookshelf container
+
+Before the application can be deployed to Container Engine, you will need build and push the image to [Google Container Registry](https://cloud.google.com/container-registry/). 
+
+    docker build -t gcr.io/your-project-id/bookshelf .
+    gcloud docker push gcr.io/your-project-id/bookshelf
+
```

Comment:
You should give instructions to update the RC image to the one you just built. Yours also doesn't work since it's non-public.